packages/common/src/project/types.ts

```diff
@@ -3,17 +3,17 @@
 
 import {SubqlKind} from './constants';
 
-export interface IProjectManifest {
+export interface IProjectManifest<M extends SubqlMapping = SubqlMapping> {
   specVersion: string;
   description: string;
   repository: string;
-  dataSources: SubqlDataSource[];
-  schema: string;
```

Comment:
I have removed it here because the type has changed from `string` to `{ file: string; }`. Each of the versioned interfaces include the specific type.

I'm happy to add it back in as a union type or ensure that its mapped to `{ file: string; }`

Reply:
We can even make the type of schema `GraphQLDocument` or something alike.